README.md

```diff
@@ -1304,10 +1304,12 @@ class {'collectd::plugin::sensors':
 class {'collectd::plugin::snmp':
   data  =>  {
     amavis_incoming_messages => {
-      'Type'     => 'counter',
-      'Table'    => false,
-      'Instance' => 'amavis.inMsgs',
-      'Values'   => ['AMAVIS-MIB::inMsgs.0']
+      'Type'        => 'counter',
+      'Table'       => false,
+      'Instance'    => 'amavis.inMsgs',
+      'Values'      => ['AMAVIS-MIB::inMsgs.0']
+      'Ignore'      => '00:00'
```

Comment:
Hi, I've taken over responsibility from @lilhuang for this.  I believe 00:00 is a good example.  In our context, it is what SNMP returns for an empty switch interface alias so we want to ignore it - it seems reasonable that others may encounter a similar situation.  I've updated the commit to README.md in the pull request to also include an example of wildcard matching a string and putting multiple Ignore values in an array.  

`+      'Ignore'      => [ '00:00', '*IgnoreString' ],`